lib/child_process.js

```diff
@@ -98,7 +98,7 @@ const MAX_BUFFER = 1024 * 1024;
  * @param {{
  *   cwd?: string;
  *   detached?: boolean;
- *   env?: Object;
+ *   env?: object;
```

Comment:
The `object` type represents all non-primitive values in TS, should we use a more precise type? (Or maybe that’s JSDoc syntax and not TypeScript and `object` means something else?)
```suggestion
 *   env?: Record<string, string>;
```

Reply:
Subsequent PR? This is just about getting our current entries to pass linting.